app/javascript/dashboard/i18n/locale/es/cannedMgmt.json

```diff
@@ -67,7 +67,7 @@
       },
       "CONFIRM": {
         "TITLE": "Confirmar eliminación",
-        "MESSAGE": "¿Está seguro de eliminar ",
+        "MESSAGE": "¿Estás seguro de eliminar ",
```

Comment:
I believe Estás is used in informal context. I don't really know spanish, but just checking whether the usage is correct.

Reply:
@pranavrajs mhmm, yes and not. I chose that one to be consistent across the whole translation. If you prefer the "formal" one, I can change all of them. i.e. Introduzca is the "formal" one, Introduce is the other one, etc.

Reply:
@RicardoRamirezR Thanks for the details. I would suggest to use the formal one if that is possible. :)

Reply:
Done